Updated VEBA Appliance deployment guides (#609)
vmware-samples/vcenter-event-broker-appliance: docs/kb/install-knative.md

> **Note:** The minimum vSphere Privileges that is required for proper VEBA UI functionality are: **Register Extension**, **Update Extension** (Installing Plugins) and **Manage Plugins** (Updating Plugins)

#### **Horizon** (**Optional**)

  * Enable Horizon Event Provider - Enable Horizon Event Provider

>   * Horizon Server - Enable Horizon Event Provider

Reviewer: Wrong description?

  * Horizon Domain Name - Active Directory Domain the username to login to the Horizon Server belongs to (e.g. corp)
  * Horizon Username - Username to login to Horizon Server (UPN-style not allowed)
  * Horizon Password - Password to login to Horizon Server
  * Disable Horizon Server TLS Verification - Disable TLS Verification for Horizon Server (required for self-sign certificate)

> **Note:** The minimum Horizon XXX

#### **Webhook** (**Optional**)

 * Enable Webhook Event Provider - Enable Webhook Event Provider
 * Basic Auth Username (Optional) - Username to login to webhook endpoint
 * Basic Auth Password (Optional) - Password to login to webhook endpoint
#### **Event Processor Configuration** (**Required**)
  * Event Processor - Choose Knative

> **Note:** Selecting Knative here will default to using the Embedded Knative deployment within the VMware Event Broker Appliance.  If this is your intent, you **do not have to fill** in any of the settings for the Knative Configuration section below. The Knative Configuration section below is for use with **an External Knative** installation.

#### **Existing Knative Environment Configuration** (**Optional**) 

> <font color="red">This configuration option has been deprecated in the VMware Event Broker Appliance v0.7 release and will be removed in next release. Embedded Knative will automatically become the future default behavior and no additional input will be required.</font>

Reviewer: Other notes use bold style

> <font color="red">This configuration option has been deprecated in the VMware Event Broker Appliance v0.7 release and will be removed in next release. Embedded Knative will automatically become the future default behavior and no additional input will be required.</font>

Reviewer: ```suggestion
<font color="red">This configuration option has been deprecated in the VMware Event Broker Appliance v0.7 release in favor of the default embedded Knative installation and thus will be removed in next release.</font>
```
